### ISSUE TITLE
Velocypack example builder recompile with -fPIC

### ISSUE THREAD

**Issue by sandrokeil** [sandrokeil/arangodb-php-driver#4]
After #3 was resolved, new error occurred.

```
/usr/bin/ld: //usr/local/lib/libvelocypack.a(Builder.cpp.o): relocation R_X86_64_TPOFF32 against `_ZGVZN8arangodb10velocypack7Builder19sortObjectIndexLongEPhRSt6vectorImSaImEEE7entries' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: //usr/local/lib/libvelocypack.a(Slice.cpp.o): relocation R_X86_64_PC32 against symbol `_ZN8arangodb10velocypack15SliceStaticData7TypeMapE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
Makefile:132: recipe for target 'arangodb.so' failed
make: *** [arangodb.so] Error 1
```

How to resolve this?

**Comment by sandrokeil** [comment]
@martin-schilling Maybe this could help [Things to remember when compiling/linking C/C++ software](https://gist.github.com/gubatron/32f82053596c24b6bec6), at least me. ;-) We should integrate the [velocypack](https://github.com/arangodb/velocypack/blob/master/examples/Embedding.md) stuff in our extension, so no shared library is needed. 

A `libvelocypack.a` (static library) file is already present. [How can we use it](https://stackoverflow.com/a/18318456) or [maybe this](https://stackoverflow.com/a/3701129) for the PHP extension?

**Comment by martin-schilling** [comment]
@sandrokeil I think i found the solution :). Try adding -fPIC to deps/velocypack/cmake/Modules/AR_CompilerSettings.cmake:10 BASE_COMPILER_OPTIONS. The whole main.cpp works for me now

**Comment by sandrokeil** [comment]
Solved by #5
